htsohm/db/material.py

```diff
@@ -18,6 +18,7 @@ class Material(Base):
     id           = Column(Integer, primary_key=True)
     run_id       = Column(String(50))
     uuid         = Column(String(40))
+    parent       = Column(String(40))
```

Comment:
This commit should really be divided into 2 commits: (1) reimplementation of mutation code, and (2) modifications for convex hull method. All the mutation code was pulled from another branch, right? The way it is now, it is hard to tell what was code that was already working (and previously reviewed) and the new code that needs to be reviewed.